src/main/scala/com/cognite/sdk/scala/common/dataTypes.scala

```diff
@@ -30,15 +30,15 @@ final case class SdkException(
       .getOrElse("")
     val uriMessage = uri
       .map(u => s", in response$responseCodeMessage to request sent to ${u.toString()}")
-      .getOrElse("")
+      .getOrElse(responseCodeMessage)
     val requestIdMessage = requestId.map(id => s", with request id $id").getOrElse("")
     val superString = super.toString
     val superMessage = if (superString.length > 0) {
       superString
     } else {
       "Missing error message"
     }
-    s"$superMessage$uriMessage$responseCodeMessage$requestIdMessage"
+    s"$superMessage$uriMessage$requestIdMessage"
```

Comment:
No whitespaces in between?

Reply:
No, white space and commas are included in the substrings themselves.